installed webpack-cli will output a error while process exit

<!-- Please don't delete this template because we'll close your issue -->
<!-- Before creating an issue please make sure you are using the latest version of webpack. -->

# Bug report
When webpack is installed but webpack-cli is not . After put yes to shell, at last it will install webpack-cli ,but still output an error to shell .
<!-- Please ask questions on StackOverflow or the webpack Gitter. -->
<!-- https://stackoverflow.com/questions/ask?tags=webpack -->
<!-- https://gitter.im/webpack/webpack -->
<!-- Issues which contain questions or support requests will be closed. -->

**What is the current behavior?**
➜  react-babel-study npm run test

> react-babel-study@1.0.0 test /Users/weiyinpeng/Documents/study-code/react-babel-study
> npm uninstall webpack-cli && npm run webp

npm WARN react-babel-study@1.0.0 No description
npm WARN react-babel-study@1.0.0 No repository field.

audited 263 packages in 2.183s

23 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> react-babel-study@1.0.0 webp /Users/weiyinpeng/Documents/study-code/react-babel-study
> webpack

{
  name: 'webpack-cli',
  package: 'webpack-cli',
  binName: 'webpack-cli',
  installed: false,
  url: 'https://github.com/webpack/webpack-cli'
}
CLI for webpack must be installed.
  webpack-cli (https://github.com/webpack/webpack-cli)

We will use "npm" to install the CLI via "npm install -D".
Do you want to install 'webpack-cli' (yes/no): y
Installing 'webpack-cli' (running 'npm install -D webpack-cli')...
npm WARN react-babel-study@1.0.0 No description
npm WARN react-babel-study@1.0.0 No repository field.

+ webpack-cli@4.1.0
added 43 packages from 26 contributors and audited 306 packages in 17.752s

28 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Error: Cannot find module 'webpack-cli'
Require stack:
- /Users/weiyinpeng/Documents/study-code/react-babel-study/node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at /Users/weiyinpeng/Documents/study-code/react-babel-study/node_modules/webpack/bin/webpack.js:120:5
    at processTicksAndRejections (internal/process/task_queues.js:93:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/weiyinpeng/Documents/study-code/react-babel-study/node_modules/webpack/bin/webpack.js'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-babel-study@1.0.0 webp: `webpack`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-babel-study@1.0.0 webp script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/weiyinpeng/.npm/_logs/2020-10-27T11_34_52_522Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-babel-study@1.0.0 test: `npm uninstall webpack-cli && npm run webp`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-babel-study@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/weiyinpeng/.npm/_logs/2020-10-27T11_34_52_548Z-debug.log

**If the current behavior is a bug, please provide the steps to reproduce.**


<!-- A great way to do this is to provide your configuration via a GitHub repository -->
<!-- The most helpful is a minimal reproduction with instructions on how to reproduce -->
<!-- Repositories with too many files or large `webpack.config.js` files are not suitable -->
<!-- Please only add small code snippets directly into this issue -->
<!-- https://gist.github.com is a good place for longer code snippets -->
<!-- If your issue is caused by a plugin or loader, please create an issue on the loader/plugin repository instead -->

**What is the expected behavior?**

expect no error while exit
<!-- "It should work" is not a helpful explanation -->
<!-- Explain exactly how it should behave -->
I will pull a request about to resolve  this problem .
https://github.com/webpack/webpack/pull/11838
It's because of require('webpack-cli') push an error to stack and it output after process exit.
**Other relevant information:**
webpack version:  5.2.0
Node.js version: v14.13.1
Operating System: mac
Additional tools:

> @yinsang Put `webpack-cli` in your package.json
I know how to fix my problem.
I mean while `questionInterface.question` has put **yes**, an error is output  after 'webpack-cli' is installed.
```
Error: Cannot find module 'webpack-cli'
```

--- a/bin/webpack.js
+++ b/bin/webpack.js
@@ -42,6 +42,19 @@ const isInstalled = packageName => {
 };
 
 /**
+ * @param {CliOption} cli options
+ * @returns {void}
+ */
+const runCli = cli => {
+	const path = require("path");
+	const pkgPath = require.resolve(`${cli.package}/package.json`);
+	// eslint-disable-next-line node/no-missing-require
+	const pkg = require(pkgPath);
+	// eslint-disable-next-line node/no-missing-require
+	require(path.resolve(path.dirname(pkgPath), pkg.bin[cli.binName]));
+};
+
+/**
  * @typedef {Object} CliOption
  * @property {string} name display name
  * @property {string} package npm package name
@@ -116,7 +129,7 @@ if (!cli.installed) {
 
 		runCommand(packageManager, installOptions.concat(cli.package))
 			.then(() => {
-				require(cli.package); //eslint-disable-line
+				runCli(cli);
 			})
 			.catch(error => {
 				console.error(error);
@@ -124,10 +137,5 @@ if (!cli.installed) {
 			});
 	});
 } else {
-	const path = require("path");
-	const pkgPath = require.resolve(`${cli.package}/package.json`);
-	// eslint-disable-next-line node/no-missing-require
-	const pkg = require(pkgPath);
-	// eslint-disable-next-line node/no-missing-require
-	require(path.resolve(path.dirname(pkgPath), pkg.bin[cli.binName]));
+	runCli(cli);
 }

